### PR TITLE
Couple of fixes to get workflows working

### PR DESCRIPTION
**Pull request by AntoineDao**
Couple of minor fixes. Tested them on workflows by creating my own docker image and running it as a new operator.

### CODE REVIEW

File: honeybee_radiance/sensorgrid.py [hunk 1]
```diff
@@ -179,8 +179,6 @@ def to_files(self, folder, count, base_name=None, mkdir=False, full_path=False):
             base_name: Optional name for base_name for sensor files (Default: self.name).
             mkdir: A boolean to indicate if the folder should be created in case it
                 doesn't exist already (Default: False).
-            full_path: A boolean to indicate if this method should return the full_path
-                to the newly created files or just the name.
 
         Returns:
             A list of filepaths to newly created files. The path can be full_path or just
```
Comment (mostaphaRoudsari) [inline honeybee_radiance/sensorgrid.py]: We need to update Returns docstring here

File: honeybee_radiance/cli/grid.py [hunk 2]
```diff
@@ -43,7 +44,8 @@ def split_grid(grid_file, count, folder, log_file):
         grid = sensorgrid.SensorGrid.from_file(grid_file)
         file_count = int(round(grid.count / count))
         files = grid.to_files(folder, file_count, mkdir=True)
-        log_file.write('\n'.join(files))
+        
+        log_file.write(json.dumps(files))
```
Comment (mostaphaRoudsari) [inline honeybee_radiance/cli/grid.py]: Curios to know why do we need json.dumps in this case?

Comment (AntoineDao) [reply honeybee_radiance/cli/grid.py]: It turns out that if you load a file as an output parameter in Argo and you then want to use that parameter to loop tasks, then the parameter must be a JSON serializable array.

ie:
## BAD
```
room_0001.pts \n
room_0002.pts 
``` 

## GOOD
```
[ "room_0001.pts", "room_0002.pts" ]
```
```
[
  {
    "name": "room_0001",
    "path": "room_0001.pts"
  },
  {
    "name": "room_0002",
    "path": "room_0002.pts"
  }
]
```

Comment (mostaphaRoudsari) [reply honeybee_radiance/cli/grid.py]: Got it! Thanks. This might be a better practice anyways.

**Pull request merged**